klpbuild/config.py

```diff
@@ -29,13 +30,19 @@
 
 class Config:
     def __init__(self, lp_name, lp_filter, kdir=False, data_dir=None, skips="", working_cs={}):
-        work_dir = os.getenv("KLP_WORK_DIR")
-        if not work_dir:
-            raise ValueError("KLP_WORK_DIR should be defined")
+        home = os.getenv("HOME")
+        if not home:
+            raise ValueError("HOME env not defined")
 
-        work = Path(work_dir)
-        if not work.is_dir():
-            raise ValueError("Work dir should be a directory")
+        self.user_conf_file = Path(home, ".config/klp-build/config")
```

Comment:
I believe that we can expect users to have HOME defined? I mean, the majority of people use the most common distros, and I have never saw anyone without HOME being defined :)

Reply:
It's still the same as the older version. You can use pathlib's Path().home() to get the home, and I think that we don't need to check if it is defined or not...